src/www/services_dhcp.php

```diff
@@ -890,7 +890,7 @@ function show_netboot_config() {
 										<td width="22%" valign="top" class="vncell"><?=gettext("Failover peer IP:");?></td>
 										<td width="78%" class="vtable">
 											<input name="failover_peerip" type="text" class="form-control host" id="failover_peerip" size="20" value="<?=htmlspecialchars($pconfig['failover_peerip']);?>" /><br />
-											<?=gettext("Leave blank to disable.  Enter the interface IP address of the other machine.  Machines must be using CARP. Interface's advskew determines whether the DHCPd process is Primary or Secondary. Ensure one machine's advskew<20 (and the other is >20).");?>
+											<?=htmlspecialchars(gettext("Leave blank to disable.  Enter the interface IP address of the other machine.  Machines must be using CARP. Interface's advskew determines whether the DHCPd process is Primary or Secondary. Ensure one machine's advskew<20 (and the other is >20)."));?>
 										</td>
```

Comment:
Maybe we should look at the use of "htmlspecialchars(gettext(" in a bit wider context for 16.7, almost all occurrences of gettext should be escaped because we don't know the end result.... maybe there's an easy way to wrap this for all frontend outputs.